kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt

```diff
@@ -451,6 +455,30 @@ interface ViewPagerAssertions : BaseAssertions {
 }
 
 /**
+ * Provides assertios for RatingBar
+ */
+interface RatingBarAssertions : BaseAssertions {
+    /**
+     *  Checks If rating bar is have number of rating as expected
+     *  @param number rating as expected
+     */
+    fun expectedNumberOfStarAs(number: Float) {
+        view.check(ViewAssertions.matches(object : TypeSafeMatcher<View>() {
```

Comment:
We put all of the custom matchers to `Matchers.kt` and declare them as classes, so that library consumers can have access to these matchers.

Reply:
Also, consider extending `BoundedMatcher<V, T>()` so that you won't have the need to check and cast the view to `RatingBar`